ush/ioda/bufr2ioda/bufr2ioda_satwind_amv_goes.py

```diff
@@ -305,7 +258,7 @@ def bufr_to_ioda(config, logger):
                 satellite_id = satellite_info["satellite_id"]
                 satellite_name = satellite_info["satellite_name"]
                 satinst = sensor_name.lower()+'_'+satellite_name.lower()
-                logger.debug("Split data for {satinst} satid = {sat}")
+                logger.debug('Split data for {satinst} satid = {sat}')
```

Comment:
this is missing a `f` so it will print literally:
`Split data for {satinst} satid = {sat}`
and not the value in the loop

`logger.debug(f'Split data for {satinst} satid = {sat}')`